### PR TITLE
Create Custom Table with plugin

### PR DESCRIPTION
**Pull request by abhaydubey1984**

### CODE REVIEW

File: wp-content/themes/twentyseventeen/Custom_Template.php [hunk 1]
```diff
@@ -0,0 +1,3 @@
+<?php
+//Template Name : Custom_Template
```
Comment (ajeets1971) [inline wp-content/themes/twentyseventeen/Custom_Template.php:2]: why you have added empty file?

File: wp-content/plugins/Custom_Contact_Form/Custom_Contact_Form.php [hunk 2]
```diff
@@ -0,0 +1,28 @@
+<?php
+/*
+Plugin Name: Contact Table
+Description: Contact Information.
```
Comment (ajeets1971) [inline wp-content/plugins/Custom_Contact_Form/Custom_Contact_Form.php:4]: remove comment

File: wp-content/themes/twentyseventeen/Custom_Template.php (hunk 1 above)
Comment (abhaydubey1984) [reply wp-content/themes/twentyseventeen/Custom_Template.php:2]: This file is need when we create custom template page so we need to write in comment for activate this page

File: wp-content/plugins/Custom_Contact_Form/Custom_Contact_Form.php (hunk 2 above)
Comment (abhaydubey1984) [reply wp-content/plugins/Custom_Contact_Form/Custom_Contact_Form.php:4]: Comment is need for custom plugin to  activate.

**Pull request merged**